desktop/components/commercial_filter/test/models/filter.coffee

```diff
@@ -28,7 +28,7 @@ describe 'Filter', ->
       query.should.not.containEql 'fragment artist on Artist'
 
     it 'includes the artist fragment and merchandisable artists fragment if that aggregation is specified', ->
-      params = new @Params({}, {})
+      params = new @Params(aggregations: ['MERCHANDISABLE_ARTISTS'], {})
```

Comment:
might be worth keeping in to demonstrate that this aggregation is still available to the front-end.